test/libsolidity/semanticTests/array/copying/copy_dynamically_sized_storage_array.sol

```diff
@@ -0,0 +1,27 @@
+contract C {
+  function() external [] s0;
+  function() external [] s1;
+  function copyDynamicallySizedStorageArray() public {
+    s0 = s1;
+  }
+
+}
+contract copyDynamicArrays {
+  C public testContract;
+
+  constructor() {
+    testContract = new C();
+  }
```

Comment:
I think we should stick to our own style guide that [recommends 4-space indents](https://docs.soliditylang.org/en/latest/style-guide.html#indentation). The two space ones are really annoying, especially with deeper nesting when you want to see which brace ends which statement.

```suggestion
contract copyDynamicArrays {
    C public testContract;

    constructor() {
        testContract = new C();
    }
```